userprofile/static/userprofile/extra_logins.js

```diff
@@ -2,16 +2,18 @@
 
   function showExtras(event) {
     event.preventDefault();
-    const extras = $("#login-box-row").find(".extra-login")
+    const extras = $("#login-box-row").find(".extra-login");
     extras.show();
     $(".show-extra-login-btn").hide();
     extras.find(":input:visible").first().focus() // focus for keyboard navigation
```

Comment:
You added the missing semicolons, so maybe you can add the missing one here.

Reply:
Thanks, I added the semicolon, squashed and rebased this!